src/act_cell.py

```diff
@@ -38,14 +38,19 @@ def state_size(self):
 
     def __call__(self, inputs, state, timestep=0, scope=None):
 
+        self.state_is_tuple = False
```

Comment:
Hmm, can you elaborate on this?

Would you propose we somehow access the state_is_tuple field on the `self.cell` object?